composer.json

```diff
@@ -48,7 +48,11 @@
         "doctrine/orm": "The Doctrine ORM integration is optional in the bundle."
     },
     "autoload": {
-        "psr-4": { "Doctrine\\Bundle\\DoctrineBundle\\": "" }
+        "psr-4": { "Doctrine\\Bundle\\DoctrineBundle\\": "" },
+        "exclude-from-classmap": ["Tests/"]
```

Comment:
Probably better than relying on an additional composer feature that (tbh) is quite obscure...